### PR TITLE
tests: fix smoketests from sharing artifacts

### PR DESCRIPTION
**Pull request by paulirish**

### CODE REVIEW

File: lighthouse-cli/test/smokehouse/smokehouse.js [hunk 1]
```diff
@@ -49,15 +49,17 @@ function runLighthouse(url, configPath, isDebug) {
...
+  const artifactsDirectory = `./.tmp/smokehouse-artifacts-${randInt}`;
   const args = [
     'lighthouse-cli/index.js',
     url,
     `--config-path=${configPath}`,
     `--output-path=${outputPath}`,
     '--output=json',
-    `-GA=${artifactsDirectory}`,
+    `-G=${artifactsDirectory}`,
+    `-A=${artifactsDirectory}`,
```
Comment (brendankenny) [inline lighthouse-cli/test/smokehouse/smokehouse.js]: any reason to split these up

**Comment by brendankenny** [comment]
nice find

File: lighthouse-cli/test/smokehouse/smokehouse.js (hunk 1 above)
Comment (paulirish) [reply lighthouse-cli/test/smokehouse/smokehouse.js]: yup. 

> `node lighthouse-cli/index.js http://localhost:10200/seo/seo-tester.html?extra_header=link%3D%253Chttp%253A%252F%252Flocalhost%253A10200%252Fseo%252F%253E%253B%2Brel%253D%2522canonical%2522 --config-path=/Users/paulirish/code/pristine/lighthouse-pristine/lighthouse-cli/test/smokehouse/seo-config.js --output-path=smokehouse-83625.report.json --output=json -GA=./.tmp/smokehouse-artifacts-83625`

image

Comment (brendankenny) [reply lighthouse-cli/test/smokehouse/smokehouse.js]: :this_is_fine:

**Comment by brendankenny** [comment]
> Error: ENOENT: no such file or directory, open './.tmp/smokehouse-artifacts-8306/artifacts.json'

this is a problem for `yarn smoke errors`, presumably because no artifacts were written...

**Comment by brendankenny** [comment]
> this is a problem for yarn smoke errors, presumably because no artifacts were written...

oh, yes, that's correct, but also fine. It's just smokehouse itself running into this. Need something like

```js
// Artifacts are undefined if they weren't written to file (e.g. if there was an error).
let artifacts;
try {
  artifacts = JSON.parse( fs.readFileSync(`${artifactsDirectory}/artifacts.json`, 'utf8'));
} catch (e) {}
```
down on L119 of `smokehouse.js`

**Pull request merged**